cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: nvidia-gpu-device-plugin
+  namespace: kube-system
+  labels:
+    k8s-app: nvidia-gpu-device-plugin
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: nvidia-gpu-device-plugin
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: cloud.google.com/gke-accelerator
```

Comment:
Curious how is this different from using `nodeSelector` field?

Reply:
`nodeSelector` can only do `key=value` checks. I wanted to do `key exists` check (because I want to run it on nodes that have `nvidia-tesla-k80` as value or `nvidia-tesla-p100` as value or any later value we may add.)